src/popup/locales/en.json

```diff
@@ -866,7 +866,8 @@
       "backUpYourSeedPhrase": "Back up your seed phrase {0} on a piece of paper or file that no one but you will ever see. Make multiple copies. Make sure you keep your seed phrase in a safe place!",
       "inCorrectOrder": "in the correct order of 12 words",
       "toBeSureYouGotItRight": "To be sure you got it right, you should {0} clicking on the button below!",
-      "verifyYourSeedPhrase": "verify your seed phrase"
+      "verifyYourSeedPhrase": "verify your seed phrase",
+      "goHome": "Go home"
```

Comment:
Can you check this with Tsvetan? As far as I know we have different naming for such button - like on 404 page.
Also I think this should be moved to common phrases.